invenio_curations/assets/semantic-ui/js/invenio_curations/deposit/RequestOrPublishButton.js

```diff
@@ -0,0 +1,108 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2024 TU Wien.
+//
+// Invenio-Curations is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React from "react";
+import { Button, Grid, Icon, Popup } from "semantic-ui-react";
+import RequestStatusLabel from "@js/invenio_requests/request/RequestStatusLabel";
+import { PublishButton } from "@js/invenio_rdm_records";
+import PropTypes from "prop-types";
+
+export const RequestOrPublishButton = (props) => {
+  const { request, record, handleCreateRequest, handleResubmitRequest, loading } =
+    props;
+  const recordIdAvailable = record?.id != null;
+  let elem = null;
+
+  if (request) {
+    switch (request.status) {
+      case "accepted":
+        elem = <PublishButton fluid />;
+        break;
+
+      case "critiqued":
+        elem = (
+          <Button
+            onClick={handleResubmitRequest}
+            loading={loading}
+            primary
+            size="medium"
+            type="button"
+            disabled={!recordIdAvailable}
+            positive
+            icon
+            labelPosition="left"
+            fluid
+          >
+            <Icon name="paper hand outline" />
+            {"Resubmit updated record"}
```

Comment:
```suggestion
            {i18next.t("Resubmit updated record")}
```
should be translated – for some reason, for me it didn't display _any_ text if wrapped in translation during development :eyes: